assets/js/home.js

```diff
@@ -32,10 +32,18 @@ $(document).ready(function () {
     }
     $('#log').prepend($.markup('log-entry', data))
   }
+  $.get('/updated', function (response) {
```

Comment:
This does not need to be called separately, since it will be invoked with `refreshData` which is called once the markup is initialised